### PR TITLE
📦 prove code to fix hapi V15+

### PR DESCRIPTION
**Pull request by maixiaojie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,34 +9,33 @@ https://developer.github.com/v3/oauth/#web-application-flow
...
 ```sh
-npm install hapi-auth-github --save
+npm install hapi-github-auth --save
 ```
 
 ### 2. Create an App on GitHub
 
 Follow the instructions in:
-[GITHUB-APP-STEP-BY-STEP-GUIDE.md](https://github.com/dwyl/hapi-auth-github/blob/master/GITHUB-APP-STEP-BY-STEP-GUIDE.md)
+[GITHUB-APP-STEP-BY-STEP-GUIDE.md](https://github.com/maixiaojie/hapi-github-auth/blob/master/GITHUB-APP-STEP-BY-STEP-GUIDE.md)
```
Comment (iteles) [inline README.md:23]: Given this and the fact that you've deleted a lot of the readme/code, I'm going to assume you didn't mean to push this as a PR to this repo.

**Review by iteles — CHANGES_REQUESTED** [review]
@maixiaojie @wenyujie12 From the PR it looks like you didn't mean to open this as a PR to this repo but rather you wanted to publish this as a separate/different module under your username.

If you are publishing this as a module publicly, it would be nice if you could mention @dwyl in the readme to give our developers a little extra credit!

Closing. Please let me know if these assumptions are incorrect and we'll reopen and discuss 👍

**Comment by maixiaojie** [comment]
Actually，I really hope this repo can support v17+, but i dont know how to do it，because we need to be compatible with previous versions. So I temporarily supported v17+ in my repo. we can create a new repo in your org to finish it, or add a new version in this repo, and pushlish it in npm. 
Look forward to your reply！

**Comment by iteles** [comment]
@maixiaojie Ah, I see! 
Could you please [open an issue](https://github.com/dwyl/hapi-auth-github/issues) about support for v17+ so that the people who watch this repo and have an interest can also see it and we can discuss there? ☺️

**Comment by maixiaojie** [comment]
of course. and i really want to join this org. what need i do .

**Pull request closed without merging**